src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1269,6 +1269,47 @@ public Tizen.NUI.PropertyMap Placeholder
             }
         }
 
+        /// <summary>
+        /// The Ellipsis property.<br />
+        /// Enable or disable the ellipsis.<br />
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
```

Comment:
Please remove this.
You need to add this tag when your API is open to the public.
And the API level of tizen_6.5 is 9.
<since_tizen> 9 </since_tizen>

Reply:
Done.